app/[lang]/pages/onboarding-welcome/page.tsx

```diff
@@ -0,0 +1,66 @@
+'use client';
```

Comment:
Does this need to be a client-component? Is `@mui/material` a client-only library?